app/assets/stylesheets/templates/_dependents-index.scss

```diff
@@ -1,25 +1,13 @@
 .dependents-index {
-  .dependents-list {
-    padding: 2.5rem 2rem;
-    border: 2px solid black;
-    border-radius: 1rem;
-
-    @media screen and (min-width: $tablet-up) {
-      max-width: 57rem;
-    }
-  }
   .dependent {
+    display: flex;
```

Comment:
there must have been a regression on this page at some point -- i think maybe the image svg became bigger when we did a style update some time back and the image icon was sitting on top of the person's name. This update prevents that from happening.